tests/Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot

```diff
@@ -16,9 +16,10 @@ Launch Jupyterhub via Routes
    Sleep  4s
    Input Text  xpath://input[@data-test-id="item-filter"]  jupyterhub
    Wait Until Page Contains  jupyterhub  timeout=15
+   Sleep  1s
    Click Element  partial link:https://jupyterhub
    Sleep  10s
-   Switch Window  JupyterHub
+   Switch Window  NEW
```

Comment:
Is there a case in which the new tab is not called JupyterHub?

Reply:
When launching jupyterhub from route it goes to the Log in page first for me, which fails the test.